src/formatDate.js

```diff
@@ -50,7 +50,27 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = fromFormat[fromFormat.length - 1];
+  const newSeparator = toFormat[toFormat.length - 1];
```

Comment:
Use destructuring assignment to extract the last element from the array for better readability. e.g., const [newSeparator] = toFormat.slice(-1);